src/routes/[lang=lang]/catalog/bootcamps/[name]/+page.svelte

```diff
@@ -21,16 +22,26 @@
 			await logIn();
 		}
 		$createBootcampOnboardingStore.walletAddress = $user.addr;
-		getModal().open()
+		getModal().open();
 	}
 </script>
 
 <ContentIntro overview={data.overview} showBreadcrumbs={true}>
-	{#if $user.loggedIn && data.signUps.includes($user.addr)}
-		<Button size="large" width="extended" state="disabled">Enrolled</Button>
-	{:else}
-		<Button size="large" width="extended" on:click={openModal}>Enroll</Button>
-	{/if}
+	<div class="row-6">
+		{#if $user.loggedIn && data.signUps.includes($user.addr)}
+			<Button size="large" width="extended" state="disabled">Enrolled</Button>
+		{:else}
+			<Button size="large" width="extended" on:click={openModal}>Enroll</Button>
+		{/if}
+		<Button
+			size="large"
+			type="ghost"
```

Comment:
Type transparent was the style I was thinking of!!
That's the way to take out the borders. We can try if that way looks better..

Reply:
@mateoroldos What do you think? Looks good with type= transparent
![image](https://user-images.githubusercontent.com/112862961/232786554-8c4aae92-eea9-4e03-b7c4-7b7cbeb1f928.png)

Reply:
Love it!!!